src/Statement/Statement.php

```diff
@@ -30,6 +35,26 @@ class Statement extends Claim {
 	const RANK_NORMAL = 1;
 	const RANK_DEPRECATED = 0;
 
+	/** @deprecated since 2.0 */
+	const RANK_TRUTH = 3;
```

Comment:
As already suggested in an other comment, I suggest to drop this completely, but do this in a separate pull request that can be merged before this one.